spec/dummy/config/initializers/active_record_belongs_to_required_by_default.rb

```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
+
+# Require `belongs_to` associations by default. This is a new Rails 5.0
+# default, so it is introduced as a configuration option to ensure that apps
+# made on earlier versions of Rails are not affected when upgrading.
+Rails.application.config.active_record.belongs_to_required_by_default = true if Rails.version.to_i >= 5
```

Comment:
This got real! With this line in, master presents 544 examples, 150 failures, 2 pending. That's great! :+1: